### ISSUE TITLE
[QUESTIONS] How to add IP switching capability ? (want to run on replit)  +more

### ISSUE THREAD

**Issue by Ghosty-Gigabytes** [farshadz1997/Microsoft-Rewards-bot#78]
Hi 
I wanted to host your script on replit.com in order to farm my accounts without having my local system turned on, but also wanted to add IP switching capability to increase my points, Do you know any solution for this.
(Btw your script is awesome, and very noob friendly 👍👍)

Thanks in advance

**Comment by farshadz1997** [comment]
Hi,
sorry but I don't know any way for IP switching on Replit.

**Comment by Ghosty-Gigabytes** [comment]
> were you successfull on hosting the script on replit? is it possible under free plan or do u need hacker plan

Yeah kinda successful, some times it throws some error regarding the 'browser' reference and some other references. Hosting it on replit doesn't require the hacker plan, though you will need the hack plan to run it 24x7.

**Comment by JittoJoseph** [comment]
you could use proxy instead of switching ips right?

**Comment by Ghosty-Gigabytes** [comment]
> you could use proxy instead of switching ips right?

Any thing which helps me to change the apparent location of the bot will work, do you have any suggestions for the same

**Comment by JittoJoseph** [comment]
In the accounts.json file you can specify a proxy for each account,
The format is in the readme.md file,
You can find random proxy addresses online of any specific location.

**Comment by abhish2** [comment]
> Author

yeah ive been stuck on solving browser reference error. it happens when you have to complete the search rewards. any fix for that?
